client/src/views/home/HomePage.vue

```diff
@@ -97,10 +97,24 @@ const routeWatchCancel = watchRoute(async () => {
 
 onMounted(async () => {
   storedDeviceDataDict.value = await storageManager.retrieveDevicesData();
+  document.addEventListener('keydown', function(event) {
```

Comment:
The API is not super easy to understand and can be improved.

Few things to note:
- the component shouldn't do the `document.addEventListener`, I don't want to look up the API for that thing when I want to add a shortcut.
- removing the event listener is just as bad, I don't want to repeat the same code
- the `onKeyPress` parameter is hard to understand (`combinations: Array<[string, () => Promise<void>, boolean?]>`)
- can callback functions take parameters?
- if I understand correctly, the third boolean parameter is the shift key, how about ctrl or alt?